From b12fab6d6b9ee79251bf958862db3ccfddb2aa8d Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Mon, 29 Oct 2018 14:58:30 +0200
Subject: [PATCH 0744/1051] arm64: dts: marvell: armada-8040-db-B: enable CP1
 I2C0

Set the controller frequency to 100MHz.

Change-Id: Ibcb7447933bd55419d4bb6bf61c0566412cb588f
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60580
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db-B.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
index 25f40a545f2b..d47278ee492a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
@@ -85,6 +85,11 @@
 	phy-mode = "rgmii-id";
 };
 
+&cp1_i2c0 {
+	status = "okay";
+	clock-frequency = <100000>;
+};
+
 &cp1_spi1 {
 	status = "okay";
 };
-- 
2.17.1

